README.md

```diff
@@ -1,10 +1,10 @@
-[![Build Status](https://travis-ci.org/fishman/ctags.svg?branch=master)](https://travis-ci.org/fishman/ctags)
+[![Build Status](https://travis-ci.org/exuberant-ctags/ctags.svg?branch=master)](https://travis-ci.org/exuberant-ctags/ctags)
```

Comment:
My idea was to only merge this PR after the migration is done.
This way everything is already prepared for the migration and then it's a push of a button to commit the changes.